README.md

```diff
@@ -45,134 +37,139 @@ For more information, please see the [build container wiki](https://github.com/l
 
 ![Fork the Connect repo on Github](/assets/images/help/ForkRepository.png)
 
 ### Step 3 - Branches on Repositories
 
-Each repository can have multiple branches with whatever name you choose. The Linaro Connect repository has two main branches - `develop` and `master`. The `develop` branch contains the version of the websites files that are used to build [staging.connect.linaro.org](https://staging.connect.linaro.org). The `master` branch contains the version of the websites files that are used to build [connect.linaro.org](https://connect.linaro.org). When submitting changes to the website you need to make sure you are on the correct branch. 
-
+Each repository can have multiple branches with whatever name you choose. The Linaro Connect repository has two main branches - `develop` and `master`. The `develop` branch contains the version of the websites files that are used to build [staging.connect.linaro.org](https://staging.connect.linaro.org). The `master` branch contains the version of the websites files that are used to build [connect.linaro.org](https://connect.linaro.org). When submitting changes to the website you need to make sure you are on the correct branch.
 
 ![Changing branch on your Fork](/assets/images/help/ChangingBranchOnFork.gif)
 
+### Step 4 - Make your changes and submit a pull request
 
-### Step 4 - Make your changes and submit a pull request.
-
-In order to get your changes added to the Connect website you must make changes to a file/files on **your** `fork` of the Connect repository and then submit a __pull request__ (or a `PR` which is a way to get your changes added to the main `repository` and not just your copy of the Connect website.) 
-
+In order to get your changes added to the Connect website you must make changes to a file/files on **your** `fork` of the Connect repository and then submit a __pull request__ (or a `PR` which is a way to get your changes added to the main `repository` and not just your copy of the Connect website.)
 
 ![Creating a pull request.](/assets/images/help/CreatingAPullRequest.gif)
 
 ![Making a change and creating a pull request.](/assets/images/help/CreatingAPullRequest.gif)
 
-
 #### Overriding the need for a reviewer on your pull request
 
 If you are certain that you need to merge the changes you have made and you are an Administrator of the repository you do the following to merge the PR.
 
-**Warning:** If you are an administrator and are using the above method of mergin your pr. Then please make sure that at least the tests __have passed__!
+**Warning:** If you are an administrator and are using the above method of mergin your PR, please ensure that the tests __have passed__ before force merging!
```

Comment:
"mergin" is missing a trailing 'g'.

I know it was always missing but if you are going to edit the line ...